src/System.Private.CoreLib/src/System/Threading/WaitHandle.cs

```diff
@@ -147,19 +147,21 @@ private bool WaitOneCore(int millisecondsTimeout)
         private static SafeWaitHandle[] ObtainSafeWaitHandles(
             RuntimeThread currentThread,
             WaitHandle[] waitHandles,
+            int numWaitHandles,
```

Comment:
Sorry I put the comment the wrong place, I meant to put it in WaitHandle.Windows.cs